src/app/components/AccountDetailLayout/index.tsx

```diff
@@ -0,0 +1,176 @@
+import {
+  CaretLeftIcon,
+  CrossIcon,
+  ExportIcon,
+} from "@bitcoin-design/bitcoin-icons-react/filled";
+import Header from "@components/Header";
+import IconButton from "@components/IconButton";
+import { useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import Modal from "react-modal";
+import QRCode from "react-qr-code";
+import { Outlet, useNavigate, useParams } from "react-router-dom";
+import Avatar from "~/app/components/Avatar";
+import Loading from "~/app/components/Loading";
+import TextField from "~/app/components/form/TextField";
+import api, { GetAccountRes } from "~/common/lib/api";
+import msg from "~/common/lib/msg";
+import { Account } from "~/types";
+
+type AccountAction = Pick<Account, "id" | "name">;
+
+function AccountDetailLayout() {
+  const navigate = useNavigate();
+
+  const { t } = useTranslation("translation", {
+    keyPrefix: "accounts.account_view",
+  });
+  const [account, setAccount] = useState<GetAccountRes>();
+  const { id } = useParams() as { id: string };
+  const [exportLoading, setExportLoading] = useState(false);
+  const [exportModalIsOpen, setExportModalIsOpen] = useState(false);
+  const [lndHubData, setLndHubData] = useState({
+    login: "",
+    password: "",
+    url: "",
+    lnAddress: "",
+  });
+
+  useEffect(() => {
+    (async () => {
+      const account = await api.getAccount(id);
+      setAccount(account);
+    })();
+  }, [id, navigate]);
+
+  async function exportAccount({ id, name }: AccountAction) {
```

Comment:
Isn't this only possible from the account detail page? maybe it should be in there instead.

Also, I cannot see the export functions when I add an Lndhub account, strange

![image](https://github.com/getAlby/lightning-browser-extension/assets/33993199/b68b7d5d-38cd-485d-8fa9-30e0dbc3c4e4)

Reply:
Should the lndhub-specific code be moved to a separate component?

Reply:
I think this stuff will be moved to the Account Detail page. I'd rather wait for @stackingsaunter's proposal before we put more work into creating components right now.

Reply:
@reneaaron didn't we talk about this in Telegram DM? It used to be only in the Detail Page, which was the issue with having the layout parent page. Because the layout page needs to know (e.g. by path segment) which page we are in and show the export button only for the detail page. You mentioned why should it only be in the detail page. If I remember correctly.

The problem is, this csv export button is part of the header markup. But it should only be displayed in the detail page (if this is the case). That's why I thought it might be better to have the header as a component, with a prop to configure displaying the export button. This way it can be controlled within the subpages.